metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/README.md

```diff
@@ -0,0 +1,3 @@
+# bro Parser
+
+This is the bro parser
```

Comment:
I believe that the bundle needs to be it's own thing, but I'm not 100% sure, since I never questioned it